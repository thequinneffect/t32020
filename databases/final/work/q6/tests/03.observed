tests/03.sh: 1: ./q6: Permission denied
