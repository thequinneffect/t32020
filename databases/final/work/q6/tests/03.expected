Discography for Grand Floyd
--------------------
Pump up the Ocean (2001) (post-rock)
--------------------
 1. Get out of Vision (10:39)
 2. Up to the Sky (5:52)
 3. Jump on New York (8:40)
 4. Fly to Vision (4:02)
 5. Where is Pandemic (6:02)
 6. Master of the Bed (6:05)
 7. On the Road to the Sky (8:58)
 8. Running from the Dance Floor (5:17)
 9. Take Dancing (5:59)
10. Where is the Sun (6:27)
--------------------
Get out of Blue (2002) (country)
--------------------
 1. Why the Sea (4:00)
 2. Destroy Plague (5:42)
 3. Escape from Summer (4:20)
 4. Bad for White (6:33)
 5. Jump on Dancing (1:45)
--------------------
Take the Moon (2006) (pop)
--------------------
 1. Jump on the Clouds (3:49)
 2. Take Summer (4:40)
 3. Bad for Haze (4:16)
 4. What is the Horizon (4:46)
 5. View to the Sun (5:22)
 6. Up to Pandemic (4:20)
 7. Ode to the Moon (6:33)
 8. Jump on this Night (6:08)
 9. Get out of Dancing (11:42)
10. Jump on Tokyo (4:35)
12. Fast Train to the Heat (5:02)
--------------------
Waiting for Haze (2008) (hip-hop)
--------------------
 1. Roll with the Moon (1:54)
 2. Up to this World (7:28)
 3. Run with the Ocean (5:56)
 4. Up to the Sea (2:28)
 5. Bad for the Dance Floor (8:29)
 6. Jump on Love (2:13)
 7. Escape from the Bed (1:38)
 8. Bring the Sea (6:15)
 9. Fly to the Moon (4:50)
--------------------
Fast Train to the Farm (2010) (hip-hop)
--------------------
 1. Get out of the Nights (12:19)
 2. Escape from Love (7:09)
 3. Where is Khatmandu (8:13)
 4. What about Summer (4:40)
 5. On the Road to the Moon (3:22)
 6. Force Haze (8:57)
 7. Get out of this City (6:05)
--------------------
What about this City (2011) (country)
--------------------
 1. Beyond this City (3:53)
 2. Run with Dancing (5:40)
 3. View to Haze (5:53)
 4. Where is the Clouds (3:14)
 5. Run with Summer (5:46)
 6. Up to the Moon (4:04)
